packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -28,14 +28,51 @@ import { isGuildBasedChannel } from '../type-guards';
  *
  * @example
  * ```typescript
- * const handler = new PaginatedMessage();
+ * const myPaginatedMessage = new PaginatedMessage();
+ * // Once you have an instance of PaginatedMessage you can call various methods on it to add pages to it.
+ * // For more details see each methods' documentation.
```

Comment:
```suggestion
 * // For more details see each method's documentation.
```
For the same reason you don't say each items, it's each item in a collection of items.